include/marl/scheduler.h

```diff
@@ -104,6 +104,10 @@ class Scheduler {
   MARL_EXPORT
   static Scheduler* get();
 
+  // setBound() sets the scheduler bound to the current thread.
+  MARL_EXPORT
+  static void setBound(Scheduler* scheduler);
```

Comment:
Can this be private & non-exported?

Reply:
Sure, it can be anything you want :)
As long as the MSAN error is suppressed, I have no problem making any modification to this change.

Reply:
Done. PTAL

Reply:
I just didn't know if you were calling this directly from SwiftShader.
LGTM, thank you!